src/components/select/demoSelectHeader/style.css

```diff
@@ -0,0 +1,19 @@
+.demo-header-searchbox {
+  border: none;
+  outline: none;
+  height: 100%;
+  width: 100%;
+  padding: 0;
+}
+
+.demo-select-header {
+  box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.1), 0 0 0 0 rgba(0, 0, 0,
+  0.14), 0 0 0 0 rgba(0, 0, 0, 0.12);
+  padding-left:  40px;
+  height: 48px;
+  cursor: pointer;
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: auto;
+}
```

Comment:
@DerekLouie The reason this CSS is not applied is because we prepend the `.selectdemoBasicUsage` class to the front of every selector so that it is "scoped" to only this demo.

The problem with this approach is that the menu container which houses the header is actually appended to the body, which is outside of our scoped demo. As far as I know, there is not currently a way to achieve what you want without moving your CSS somewhere more global like the site's `app.css` which I would recommend against.

If you want to do this, perhaps we can update the build system to recognize a `global-styles.css` file in the demos and append that to the generated `styles.css` without the demo scoping. We would just have to be very careful to always add some manual CSS classes (like you've done above) to ensure we don't affect other parts of the app.